CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
+## [unreleased] - yyyy-mm-dd
+
+### Changed
+* Upgraded editor version to 2022.3.7f1 (#855)
```

Comment:
Should mention the important packages that were automatically upgraded (and check their changelogs to see if there is something of note in there that we need to take into account)